### DISCUSSION TITLE
Train Mujoco model using Gymnasium API from Farama-Foundation

### DISCUSSION THREAD

**Discussion by tiago2820** [google-deepmind/mujoco#1003] [General]
Hi,

I'm a student and I'm trying to use MuJoCo to train a model using Gymnasium API on Unity.

I'm looking for some help with how to do the connection between mujoco, Gymnasium and Unity.

The main objective is to train inverted double pendulum model from mujoco using Gymnasium on Unity.

**Comment by Balint-H** [comment]
Hello, if you'd like to build scenes with Unity the simplest approach is using the ML-Agents package. here is a tutorial on setting that up with MuJoCo:

https://github.com/Balint-H/mj-unity-tutorial/tree/main

Once you make a build of your scene with the ML-Agents components, you can use the build as Gym/gymnasium environment as described here:

https://github.com/Unity-Technologies/ml-agents/blob/develop/docs/Python-Gym-API.md

Let me know if you have questions here or on the tutorial repo!
